components/Footer/Footer.css

```diff
@@ -1,10 +1,16 @@
 .Footer {
   align-items: center;
-  background-color: var(--slate);
+  background-color: var(--navyDark);
   color: var(--mist);
+  font-family: "PF Din Display", Arial, sans-serif;
```

Comment:
Instead of putting this here, go ahead and define `font-family` and `background-color` via the `footer` element within `globalStyles.css`